blade-graphics/src/vulkan/mod.rs

```diff
@@ -556,6 +557,14 @@ fn map_texture_format(format: crate::TextureFormat) -> vk::Format {
         Tf::Bc4Snorm => vk::Format::BC4_SNORM_BLOCK,
         Tf::Bc5Unorm => vk::Format::BC5_UNORM_BLOCK,
         Tf::Bc5Snorm => vk::Format::BC5_SNORM_BLOCK,
+        Tf::Bc6hUfloat => vk::Format::BC6H_UFLOAT_BLOCK,
+        Tf::Bc6hFloat => vk::Format::BC6H_SFLOAT_BLOCK,
+        Tf::Bc7Unorm => vk::Format::BC7_UNORM_BLOCK,
+        Tf::Bc7UnormSrgb => vk::Format::BC7_SRGB_BLOCK,
+        Tf::Rgb10a2Unorm => vk::Format::A2R10G10B10_UNORM_PACK32,
```

Comment:
also, this needs to be `A2B10G10R10_UNORM_PACK32`

Reply:
So a texture called Rgb should be Bgr? I'm confused.

I saw that BGR was more supported for this format on gpuinfo xD

Do you mean that the distinction is not important to the user of blade so it's better to have one and choose the most adopted?

Reply:
In vulkan, the formats that have "_PACK??" at the end are Big Endian encoded. So for all practical purposes, if you see PACK32, treat the components in the reverse order. Thus, our `Rgb10a2Unorm` would be equivalent to `A2G10B10A10_UNORM_PACK32` in Vulkan.

Reply:
Oh, how intuitive! 

Well that explains why the bgra format on gpuinfo was more supported 

Thanks for the explanation and will fix

Reply:
As for metal, it is correct as is?

Seems wgpu-hal in `metal/adapter.rs` does "Tf::Rgb10a2Uint => RGB10A2Uint,"

Reply:
Yes, Metal is a sane API :)